salt/transport/zeromq.py

```diff
@@ -392,6 +392,9 @@ async def consume(running):
                     if msg:
                         try:
                             await callback(msg)
+                        except TypeError as exc:  # pylint: disable=broad-except
```

Comment:
```suggestion
                        except TypeError as exc:
```
It's not a broad exception.